### ISSUE TITLE
ASR33

### ISSUE THREAD

**Issue by sbridger** [Swordfish90/cool-retro-term#684]
Frankly, VDU's are for pussies. 
A real mans PDP11 is connected to raw steel. 
An ASR33. 

fonts:

https://iclces.uk/articles/a_teletype_font.html
https://iclces.uk/_static/teletype.ttf

https://alt.folklore.computers.narkive.com/BTmvrTdp/teletype-asr-33-font-redux
http://web.archive.org/web/20000819043545/ourworld.compuserve.com/homepages/mzanzig/MZFUN1TT.ZIP

I'm not using Retroterm ,so this isn't a feature request, more of an FYI

**Comment by larsbrinkhoff** [comment]
Agreed, but I'd say the cool-retro-term visual effects aren't well suited for a teletype feel.  Also, how would you emit the oily smells that are vital to the experience?

**Comment by sbridger** [comment]
This is your route to monetisation: A USB hot oil & paper chads spritzer
